src/pages/signin/SignInPageLayout/Footer.tsx

```diff
@@ -164,24 +170,34 @@ function Footer(props) {
                 ) : null}
                 <View style={pageFooterWrapper}>
                     <View style={footerColumns}>
-                        {_.map(columns({navigateFocus: props.navigateFocus}), (column, i) => (
+                        {columns({navigateFocus}).map((column, i) => (
                             <View
                                 key={column.translationPath}
                                 style={footerColumn}
                             >
-                                <Text style={[styles.textHeadline, styles.footerTitle]}>{props.translate(column.translationPath)}</Text>
+                                <Text style={[styles.textHeadline, styles.footerTitle]}>{translate(column.translationPath)}</Text>
                                 <View style={[styles.footerRow]}>
-                                    {_.map(column.rows, (row) => (
-                                        <Hoverable key={row.translationPath}>
+                                    {column.rows.map(({link, onPress, translationPath}) => (
+                                        <Hoverable key={translationPath}>
                                             {(hovered) => (
                                                 <View>
-                                                    <TextLink
-                                                        style={[styles.footerRow, hovered ? styles.textBlue : {}]}
-                                                        href={row.link}
-                                                        onPress={row.onPress}
-                                                    >
-                                                        {props.translate(row.translationPath)}
-                                                    </TextLink>
+                                                    {link ? (
```

Comment:
```suggestion
                                                    {!!link ? (
```

Render conditions should be `boolean` 

(do same with `onPress`)